Sources/FluentKit/Properties/BooleanPropertyFormat.swift

```diff
@@ -69,6 +77,10 @@ public struct OneZeroBooleanPropertyFormat: BooleanPropertyFormat {
 
 extension BooleanPropertyFormat where Self == OneZeroBooleanPropertyFormat {
     public static var oneZero: Self { .init() }
+    
+    public var description: String {
```

Comment:
`extension BooleanPropertyFormat where Self == OneZeroBooleanPropertyFormat` effectively resolves to `extension OneZeroBooleanPropertyFormat` so i think it's better to just use a normal extension like `extension OneZeroBooleanPropertyFormat: CustomStringConvertible` for the conformance just to make it possibly easier for the compiler, and also make it more straight-forward for other readers.  

The reason `extension BooleanPropertyFormat where Self == OneZeroBooleanPropertyFormat` is used at all, is for some compiler-required type inference in the `static var`s, otherwise it doesn't actually do much.
For example if you try:
```swift
extension BooleanPropertyFormat {
    public static var oneZero: Self { OneZeroBooleanPropertyFormat() }
}
```
it won't work (compiler will complain). but what is defined in the code does work because of that `where` clause.